cli/crates/cli/tests/length.rs

```diff
@@ -0,0 +1,62 @@
+mod utils;
+
+use json_dotpath::DotPaths;
+use serde_json::{json, Value};
+use utils::consts::{LENGTH_CREATE_MUTATION, LENGTH_SCHEMA, LENGTH_UPDATE_MUTATION};
+use utils::environment::Environment;
+
+#[test]
+fn length() {
+    let mut env = Environment::init(4009);
```

Comment:
This should be taken as well, I think 4014 is the first free one. Best to run a search and see if it's used.